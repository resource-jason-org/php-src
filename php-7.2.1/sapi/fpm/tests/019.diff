001+ sh: 1: -F: not found
002+ 
003+ Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
004+ Ping error
005+ File php-fpm.log.tmp missing
006+ File php-fpm.acc.tmp missing
007+ File php-fpm.slw.tmp missing
008+ File php-fpm.pid.tmp missing
001- [%s] NOTICE: fpm is running, pid %d
002- [%s] NOTICE: ready to handle connections
003- Ping ok
004- File php-fpm.log.tmp exists
005- File php-fpm.acc.tmp exists
006- File php-fpm.slw.tmp exists
007- File php-fpm.pid.tmp exists
008- [%s] NOTICE: Terminating ...
009- [%s] NOTICE: exiting, bye-bye!
010+ 
011- 127.0.0.1 -  %s "GET /ping" 200
011+ Warning: readfile(/home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.acc.tmp): failed to open stream: No such file or directory in /home/php-src/php-7.2.1/sapi/fpm/tests/019.php on line 48