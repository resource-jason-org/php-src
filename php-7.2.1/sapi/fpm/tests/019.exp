[%s] NOTICE: fpm is running, pid %d
[%s] NOTICE: ready to handle connections
Ping ok
File php-fpm.log.tmp exists
File php-fpm.acc.tmp exists
File php-fpm.slw.tmp exists
File php-fpm.pid.tmp exists
[%s] NOTICE: Terminating ...
[%s] NOTICE: exiting, bye-bye!
File php-fpm.pid.tmp removed
127.0.0.1 -  %s "GET /ping" 200