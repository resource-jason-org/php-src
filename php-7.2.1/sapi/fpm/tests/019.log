
---- EXPECTED OUTPUT
[%s] NOTICE: fpm is running, pid %d
[%s] NOTICE: ready to handle connections
Ping ok
File php-fpm.log.tmp exists
File php-fpm.acc.tmp exists
File php-fpm.slw.tmp exists
File php-fpm.pid.tmp exists
[%s] NOTICE: Terminating ...
[%s] NOTICE: exiting, bye-bye!
File php-fpm.pid.tmp removed
127.0.0.1 -  %s "GET /ping" 200
---- ACTUAL OUTPUT
sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Ping error
File php-fpm.log.tmp missing
File php-fpm.acc.tmp missing
File php-fpm.slw.tmp missing
File php-fpm.pid.tmp missing
File php-fpm.pid.tmp removed

Warning: readfile(/home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.acc.tmp): failed to open stream: No such file or directory in /home/php-src/php-7.2.1/sapi/fpm/tests/019.php on line 48
---- FAILED
