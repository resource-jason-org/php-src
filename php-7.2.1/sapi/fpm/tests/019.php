<?php

include "include.inc";

$logfile = 'php-fpm.log.tmp';
$accfile = 'php-fpm.acc.tmp';
$slwfile = 'php-fpm.slw.tmp';
$pidfile = 'php-fpm.pid.tmp';
$port = 9000+PHP_INT_SIZE;

$cfg = <<<EOT
[global]
error_log = $logfile
pid = $pidfile
[test]
listen = 127.0.0.1:$port
access.log = $accfile
slowlog = $slwfile;
request_slowlog_timeout = 1
ping.path = /ping
ping.response = pong
pm = dynamic
pm.max_children = 5
pm.start_servers = 2
pm.min_spare_servers = 1
pm.max_spare_servers = 3
EOT;

$fpm = run_fpm($cfg, $tail, '--prefix '.__DIR__);
if (is_resource($fpm)) {
    fpm_display_log($tail, 2);
    try {
		run_request('127.0.0.1', $port);
		echo "Ping ok\n";
	} catch (Exception $e) {
		echo "Ping error\n";
	}
	printf("File %s %s\n", $logfile, (file_exists(__DIR__.'/'.$logfile) ? "exists" : "missing"));
	printf("File %s %s\n", $accfile, (file_exists(__DIR__.'/'.$accfile) ? "exists" : "missing"));
	printf("File %s %s\n", $slwfile, (file_exists(__DIR__.'/'.$slwfile) ? "exists" : "missing"));
	printf("File %s %s\n", $pidfile, (file_exists(__DIR__.'/'.$pidfile) ? "exists" : "missing"));

	proc_terminate($fpm);
    fpm_display_log($tail, -1);
    fclose($tail);
    proc_close($fpm);
	printf("File %s %s\n", $pidfile, (file_exists(__DIR__.'/'.$pidfile) ? "still exists" : "removed"));
	readfile(__DIR__.'/'.$accfile);
}

?>
