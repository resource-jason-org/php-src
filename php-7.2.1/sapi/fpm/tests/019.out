sh: 1: -F: not found

Warning: fsockopen(): unable to connect to 127.0.0.1:9008 (Connection refused) in /home/php-src/php-7.2.1/sapi/fpm/tests/fcgi.inc on line 255
Ping error
File php-fpm.log.tmp missing
File php-fpm.acc.tmp missing
File php-fpm.slw.tmp missing
File php-fpm.pid.tmp missing
File php-fpm.pid.tmp removed

Warning: readfile(/home/php-src/php-7.2.1/sapi/fpm/tests/php-fpm.acc.tmp): failed to open stream: No such file or directory in /home/php-src/php-7.2.1/sapi/fpm/tests/019.php on line 48